test/WebSites/FiltersWebSite/Controllers/FilterPipelineController.cs

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace FiltersWebSite
+{
+    [SkipGlobalFilters]
+    public class FilterPipelineController : Controller
```

Comment:
Here's what I'd like to see in this area:
1. Add some **simple** functional tests that cover user scenarios in BasicWebSite
2. Delete FiltersWebSite

Reply:
I'll do that in a follow up. There's some unrelated useful tests in there - like the middleware as a filter test and some auth stuff - and it would detract from this test.